.changeset/kind-numbers-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+adding feature flag for LogBroadcaster #added
```

Comment:
This should be internal; we do not need NOPs to act on this.

Reply:
I'm not sure that is the right standard. This is public user-facing configuration that needs to be documented. We should include the fully qualified config field path here too.